### ISSUE TITLE
Clearing the form field

### ISSUE THREAD

**Issue by Koushikphy** [m3uzz/select_form_field#16]
How do you clear the select form field. I'm using a `TextEditingController` and calling `clear` method to it doesn't clear the form field

**Comment by ehusmann** [comment]
It seems like the field does not listen correctly to the controller. When you try the opposite, setting a value through the controller, it also does not work. It will change the selected item in the drop down list, but it does not change the value displayed in the field. In other words, it is not possible right now to change the value through programmatic action.

**Comment by hslbetto** [comment]
Sorry, I don't know if I understood fine.
Check the example app inside this package, there I use controller clear() and it work fine.
https://github.com/m3uzz/select_form_field/blob/master/example/lib/main.dart

**Comment by ehusmann** [comment]
I have just tested that and it does indeed work (both with the clearing of the field, as well as setting it programmatically). However, it works when you put the element inside of a stateful widget and make the change in the controller in the setState method. That seems to me like a wrong thing to do. My expectation is that when I change the value in the controller, it automatically updates the input element (as it does in normal TextFields for example). 

In other words, I would prefer to see a solution where the select form field implements the listening to the controller and updating the state, in stead of me having to do that. But, having said that, it does work like this.
